DUI3-DX/Sdk/Speckle.Converters.Common.DependencyInjection/RawConversionRegisterer.cs

```diff
@@ -40,4 +41,27 @@ private static void RegisterRawConversionsForType(ContainerBuilder containerBuil
         .InstancePerLifetimeScope();
     }
   }
+
+  public static ContainerBuilder RegisterTypesInAssemblyAsInterface(
+    this ContainerBuilder containerBuilder,
+    Assembly assembly,
+    Type interfaceType
+  )
+  {
+    bool isGeneric = interfaceType.IsGenericType;
+    foreach (var type in assembly.GetTypes().Where(type => type.IsClass && !type.IsAbstract))
+    {
+      IEnumerable<Type> interfaceTypes = type.GetInterfaces();
+      interfaceTypes = isGeneric
+        ? interfaceTypes.Where(it => it.IsGenericType && it.GetGenericTypeDefinition() == interfaceType)
+        : interfaceTypes.Where(it => it == interfaceType);
+
+      foreach (var iterfaceTypeMatch in interfaceTypes)
+      {
+        containerBuilder.RegisterType(type).As(iterfaceTypeMatch).InstancePerLifetimeScope();
+      }
+    }
+
+    return containerBuilder;
+  }
```

Comment:
Neat, my RegisterRawConversion functions ran through all assemblies, which at the time I already considered somewhat unsafe. Loading from specified assemblies is so much better and gives us somewhat control over what we want to load up.